### PR TITLE
Full logo, font, uuid

### PR DESCRIPTION
**Pull request by jpadams**
@maju6406 I really like what you did. Much simpler. I still wanted full logo with puppet name in it, so I made this PR to add that and make all fonts san-serif. I also added another field of the VM uuid, which we'll be tracking to show that we have VMware metadata as a fact as well.

### CODE REVIEW

File: site/profile/files/sample_website/css/main.css [hunk 1]
```diff
@@ -1,18 +1,23 @@
-body { margin: 60px auto; width: 70%; background:black url(../img/logo.png) no-repeat left top;
+body {
+  margin-top: 200px;
+  margin-left: 60px;
+  width: 70%;
+  font-family: 'Helvetica', 'Arial', 'Sans-Serif';
+  background:black url(../img/logo.png) no-repeat left top;
```
Comment (jpadams) [inline site/profile/files/sample_website/css/main.css:6]: @maju6406 I just consolidated the font-family in body so it affects all child elements.

File: site/profile/templates/index.html.epp [hunk 2]
```diff
@@ -17,15 +17,18 @@
               <img src="img/apple.png" height="128" width="128" />
             <% } -%>
             <h1>System Info </h1>
-            <p>FQDN :
-                <%= $fqdn %>
+            <p>fqdn/certname:
+                <%= $trusted[certname] %>
```
Comment (jpadams) [inline site/profile/templates/index.html.epp:21]: @maju6406 I'm drawing fqdn from trusted certname to highlight concept of certificate auth and fact that our certnames are based on fqdn in this environment.

**Comment by maju6406** [comment]
@jpadams Adding a VM UUID is a good call. All the other changes sound good to me. Should I merge?

**Comment by jpadams** [comment]
@maju6406 +1 merge away

**Pull request merged**